consensus/enclave/mock/src/lib.rs

```diff
@@ -351,7 +351,7 @@ impl ConsensusEnclave for ConsensusServiceMockEnclave {
         };
 
         let timestamp = if block_version.timestamps_are_supported() {
-            parent_block.timestamp + 1
+            inputs.timestamp
         } else {
             0
         };
```

Comment:
Note this is ignoring the inputs.timestamp for non timestamp version, but it's similar to the enclave impl, the worker.rs populates the value and it doesn't have knowledge of the supported block version.